src/components/Popover/index.js

```diff
@@ -8,16 +9,32 @@ import withWindowDimensions from '../withWindowDimensions';
  * This is a convenience wrapper around the Modal component for a responsive Popover.
  * On small screen widths, it uses BottomDocked modal type, and a Popover type on wide screen widths.
  */
-const Popover = props => (
-    <Modal
-        type={props.isSmallScreenWidth ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED : CONST.MODAL.MODAL_TYPE.POPOVER}
-        popoverAnchorPosition={props.isSmallScreenWidth ? undefined : props.anchorPosition}
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...props}
-        animationIn={props.isSmallScreenWidth ? undefined : props.animationIn}
-        animationOut={props.isSmallScreenWidth ? undefined : props.animationOut}
-    />
-);
+const Popover = (props) => {
+    if (!props.fullscreen && !props.isSmallScreenWidth) {
+        return createPortal(
+            <Modal
+                type={CONST.MODAL.MODAL_TYPE.POPOVER}
+                popoverAnchorPosition={props.anchorPosition}
```

Comment:
Hmm now that this prop is set conditionally below (from this PR: https://github.com/Expensify/Expensify.cash/pull/3558), should we do the same here?
```suggestion
                popoverAnchorPosition={props.isSmallScreenWidth ? undefined : props.anchorPosition}
```

Reply:
No as here this will never render on mobile. @beamanator

Reply:
Ok because on mobile `props.isSmallScreenWidth` will always be `true` 👍